src/map/script.c

```diff
@@ -23099,550 +23176,556 @@ void script_run_item_unequip_script(struct map_session_data *sd, struct item_dat
 #define BUILDIN_DEF_DEPRECATED(x,args) { buildin_ ## x , #x , args, true }
 #define BUILDIN_DEF2_DEPRECATED(x,x2,args) { buildin_ ## x , x2 , args, true }
 void script_parse_builtin(void) {
-	struct script_function BUILDIN[] = {
```

Comment:
please move this formatting changes into separate commits.
Also look like you replaced tabs to spaces? this is wrong.